fastly-compute/geoip/.secret.json

```diff
@@ -0,0 +1,3 @@
+{
+ "token" : "sup3rs3cr3t"
```

Comment:
Based on the code changes in src/index.js I think the key in this should be the shared secret, no?